lib/cell.js

```diff
@@ -0,0 +1,22 @@
+module.exports = class Cell {
+  constructor(position) {
+    this.position = position;
```

Comment:
This is interesting: this bit of data (position) seems to be redundant, as you can (and do, in the board) infer a cell's position from its index in the array it is in. Looking at this class, position is set at construction but then never touched.